jmxtrans-core/src/main/java/com/googlecode/jmxtrans/model/Server.java

```diff
@@ -221,11 +239,6 @@ public String getUrl() {
 	}
 
 	@JsonIgnore
-	public JMXServiceURL getJmxServiceURL() throws MalformedURLException {
```

Comment:
Can do. However, the exception will change since I will need to throw IOException, AttachNotSupportedException, AgentLoadException, AgentInitializationException